lib/util/package-rules.js

```diff
@@ -174,13 +175,13 @@ function matchesRule(inputConfig, packageRule) {
     positiveMatch = true;
   }
   if (matchCurrentVersion) {
-    const { matches, isVersion } = versioning.get(versionScheme);
```

Comment:
Please do not remove object destructuring. see [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)